bundles/org.openhab.binding.miio/src/main/resources/database/zhimi.heater.zb1-miot.json

```diff
@@ -170,20 +229,24 @@
 				"piid": 7,
 				"friendlyName": "Private-Service - Use Time",
 				"channel": "use-time",
-				"channelType": "time",
-				"type": "Number:Time",
+				"type": "Number:Duration",
```

Comment:
I don't think that Number:Duration is a valid type. Was this something recently added to core?

Reply:
You are probaly right..
I did see it in few other bindings
https://github.com/openhab/openhab-addons/search?l=XML&q=number%3Aduration
https://github.com/openhab/openhab-addons/blame/main/bundles/org.openhab.binding.teleinfo/README.md#L83

I'll change it...